src/views/Package/PackageHeader/Details.tsx

```diff
@@ -37,15 +38,21 @@ const WithLabel: FunctionComponent<{ label: ReactNode }> = ({
 );
 
 const Downloads: FunctionComponent<{ downloads: number }> = ({ downloads }) => (
-  <Stack align="center" direction="row" spacing={1}>
-    <DownloadIcon color="gray.700" />
-    <Text>
-      <Box as="span" color="blue.500">
-        {downloads.toLocaleString()}
-      </Box>{" "}
-      Downloads
-    </Text>
-  </Stack>
+  <Tooltip
+    hasArrow
+    label="Weekly downloads are sourced from NPM"
+    placement="left"
+  >
+    <Stack align="center" direction="row" spacing={1}>
+      <DownloadIcon color="gray.700" />
+      <Text>
+        <Box as="span" color="blue.500">
+          {downloads.toLocaleString()}
+        </Box>{" "}
```

Comment:
Coloring this as blue makes the download counts look like it is a link - I think it was requested somewhere to change this (can't find the issue at the moment)

Reply:
Yeah I'm going to actually touch up these details to remain as consistent as possible with card details